.github/workflows/cibuildwheel.yml

```diff
@@ -146,39 +147,39 @@ jobs:
           path: dist/
 
       - name: Build wheels for CPython 3.12
-        uses: pypa/cibuildwheel@ce3fb7832089eb3e723a0a99cab7f3eaccf074fd # v2.16.5
+        uses: pypa/cibuildwheel@ce3fb7832089eb3e723a0a99cab7f3eaccf074fd  # v2.16.5
```

Comment:
Just want to double check that this change does not interfere with dependabot...

my guess is that it is not a problem for dependabot directly... but that it is possible/likely that dependabot will not place two spaces when it updates, and thus cause linting failures as a result.

Reply:
Looking at [the implementation commit](https://github.com/dependabot/dependabot-core/commit/b4112ce4639d7eed1e3b2e0792eb7533f7cb125f), the [comment regex](https://github.com/dependabot/dependabot-core/commit/b4112ce4639d7eed1e3b2e0792eb7533f7cb125f#diff-e610ea8f33cbfe74ada7eb6fbeab2afe2e2dad154e191bb209fb69f6ca3d7137R77) is `(?<comment>\s+#.*)`, so it should be fine with any number of leading spaces/tabs. Then [updating the version](https://github.com/dependabot/dependabot-core/commit/b4112ce4639d7eed1e3b2e0792eb7533f7cb125f#diff-e610ea8f33cbfe74ada7eb6fbeab2afe2e2dad154e191bb209fb69f6ca3d7137R101) is only replacing old with new.

There's also a [test case](https://github.com/dependabot/dependabot-core/blame/main/github_actions/spec/fixtures/workflow_files/pinned_sources_version_comments.yml#L12) with multiple spaces before and after the `#`.